### PR TITLE
Fixes audiostream recipe on Python3

### PR DESCRIPTION
**Pull request by misl6**
This PR adds python3 support to audiostream recipe.
Before merging we need to merge kivy/audiostream#24 .

### CODE REVIEW

**Comment by misl6** [comment]
With kivy/audiostream#24 that is been merged, now for me it's ok to merge also this recipe.

File: pythonforandroid/recipes/audiostream/__init__.py [hunk 1]
```diff
@@ -18,12 +18,14 @@ def get_recipe_env(self, arch):
...
             env['USE_SDL2'] = 'True'
-            env['SDL2_INCLUDE_DIR'] = '/home/kivy/.buildozer/android/platform/android-ndk-r9c/sources/android/support/include'
+            env['SDL2_INCLUDE_DIR'] = join(self.ctx.bootstrap.build_dir, 'jni', 'SDL', 'include')
 
         env['CFLAGS'] += ' -I{jni_path}/{sdl_include}/include -I{jni_path}/{sdl_mixer_include}'.format(
                               jni_path=join(self.ctx.bootstrap.build_dir, 'jni'),
                               sdl_include=sdl_include,
                               sdl_mixer_include=sdl_mixer_include)
+        env['NDKPLATFORM'] = self.ctx.ndk_platform
+        env['LIBLINK'] = 'NOTNONE'
```
Comment (AndreMiras) [inline pythonforandroid/recipes/audiostream/__init__.py]: Can you tell us about that line?

Comment (misl6) [reply pythonforandroid/recipes/audiostream/__init__.py]: Audiostream setup.py checks for env variable 'LIBLINK' in order to set android as build target.

**Comment by KeyWeeUsr** [comment]
@AndreMiras https://github.com/kivy/audiostream/blob/d42315ed8d870c6b23dae13841215cddc3ca3dc6/setup.py#L12

**Pull request merged**